sebs/gcp/storage.py

```
@@ -12,104 +12,254 @@
+        # This method could enforce SeBS specific parts or global uniqueness checks if needed.
+        # For now, assuming names passed are either compliant or corrected by `_create_bucket`.
         return name
 
-    def _create_bucket(self, name, buckets: List[str] = [], randomize_name: bool = False):
-        found_bucket = False
-        for bucket_name in buckets:
-            if name in bucket_name:
-                found_bucket = True
-                break
+    def _create_bucket(
+        self, name: str, buckets: List[str] = [], randomize_name: bool = False
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)